lib/composer_element_template.ex

```diff
@@ -0,0 +1,127 @@
+
+
+defmodule SimpleComposerDemo.Elements.Composer do
+  @moduledoc """
+  Element responsible for placing first recived frame above the other and sending forward
+  buffor with merged frame binary in payload.
+  handle_init: initialize two Erlang :queue-s,
+  handle_process:
+    1. puts recived frame and saves it to appropriate queue (based on from with input pad it recived frame)
+    2. checks whether there are elements in both queues, if so it runs merge_frames(first_frame_binary, second_frame_binary) function
+    3. wraps binary of merged frame into buffer
+    4. sends forward (to encoder or sink pad) wraped buffer
+  handle_caps: handle caps differences between input pads and output pad
+  """
+
+  use Membrane.Filter
+  alias Membrane.RawVideo
+  alias Membrane.Buffer
+
+  def_input_pad(:first_input,
+                demand_unit: :buffers,
+                demand_mode: :auto,
+                caps: {RawVideo, pixel_format: :I420})
+
+  def_input_pad(:second_input,
+                demand_unit: :buffers,
+                demand_mode: :auto,
+                caps: {RawVideo, pixel_format: :I420})
+
+  def_output_pad(:output,
+                demand_unit: :buffers,
+                demand_mode: :auto,
+                caps: {RawVideo, pixel_format: :I420})
+
+  '''
+  def_options input_width: [
+                type: :int,
+                description: "Width of input frames (must be the same for both frames)",
+                default: 1280
+              ],
+              input_height: [
+                type: :int,
+                description: "Height of input frames (must be the same for both frames)",
+                default: 720
+              ],
+              input_bands: [
+                type: :int,
+                description: "Bands of input frames (must be the same for both frames)",
+                default: 3
+              ]
+  '''
+
+  @impl true
+  def handle_init(_options) do
+    state = %{frames_queues: %{first_queue: :queue.new(), second_queue: :queue.new()}}
+    {:ok, state}
+  end
+
+  @impl true
+  def handle_process(pad, buffer, _contex, state) do
```

Comment:
```suggestion
  def handle_process(pad, buffer, _contex, %{frames_queues: frames_queues} = state) do
```
One match less in the body of function :smile: